ui/src/main/webapp/src/app/project/project-list.component.html

```diff
@@ -33,16 +33,15 @@ <h1 i18n="Project page header">Projects</h1>
 
         <div *ngIf="projects?.length > 0" class="list-group list-view-pf projects-list">
             <div *ngFor="let project of projects"
-                class="list-group-item  project-info  tile-click"
-                (click)="viewProject($event, project)">
+                class="list-group-item  project-info  tile-click">
                 <div class="list-view-pf-actions">
                     <span class="fa-lg project-action-item" i18n-title="Edit project" title="Edit project">
                         <a (click)="editProject($event, project)" class="action-button">
                             <i class="fa fa-pencil"></i>
                         </a>
                     </span>
                     <span class="fa-lg project-action-item" i18n-title="Delete project" title="Delete project">
-                        <a (click)="deleteProject($event, project)"  class="action-button">
+                        <a (click)="$event.stopPropagation(); deleteProjectModal.show($event, project)"  class="action-button">
```

Comment:
Why you don't use `confirmDeleteProject` method?

Btw you, who criticize having too much logic in templates, are putting this `$event.stopPropagation` here? :)